src/picongpu/include/plugins/adios/ADIOSWriter.hpp

```diff
@@ -753,12 +768,12 @@ class ADIOSWriter : public ILightweightPlugin
         adiosCountParticles(threadParams, std::string());
 
         /* allocate buffer in MB according to our current group size */
-        if (!threadParams->adiosBufferInitialized)
-        {
-            ADIOS_CMD(adios_allocate_buffer(ADIOS_BUFFER_ALLOC_NOW,
-                    1.5 * ceil((double)(threadParams->adiosGroupSize) / (1024.0 * 1024.0))));
-            threadParams->adiosBufferInitialized = true;
-        }
+        /* `1 + mem` minimum 1 MiB that we can write attributes on empty GPUs */
```

Comment:
feedback from the devs: that function is quite useless by now, the real allocations take place at `adios_group_size()`